YouTube.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
 
+# Depends upon:
```

Comment:
Although obvious, this makes it easier on a user trying to debug why the script won't work.